src/pmpo_createTestMPMesh.cpp

```diff
@@ -245,8 +245,8 @@ Mesh* replicateMesh(Mesh* mesh, int replicateFactor){
     const int nCells = nCells_size*replicateFactor;
     const int nVertices = nVertices_size*replicateFactor;
      
-    DoubleVec3dView v_array = mesh->getMeshField<polyMPO::MeshF_VtxCoords>(); 
-    DoubleVec3dView vtxCoords("verticesCoordinates", nVertices);
+    auto v_array = mesh->getMeshField<polyMPO::MeshF_VtxCoords>(); 
+    MeshFView<polyMPO::MeshF_VtxCoords> vtxCoords("verticesCoordinates", nVertices);
```

Comment:
Again, I don't understand this line: type. Need to check/discuss.

and in above, we didn't use 'polyMPO::'